specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/azureFirewall.json

```diff
@@ -403,6 +427,18 @@
         "threatIntelMode": {
           "description": "The operation mode for Threat Intelligence.",
           "$ref": "#/definitions/AzureFirewallThreatIntelMode"
+        },
+        "virtualHub": {
+          "$ref": "./network.json#/definitions/SubResource",
+          "description": "The virtualHub to which the firewall belongs."
+        },
+        "firewallPolicy": {
+          "$ref": "./network.json#/definitions/SubResource",
+          "description": "The firewallPolicy associated with this azure firewall."
+        },
+        "hubIpAddresses": {
```

Comment:
Judging by example, is this a read-only property?

Reply:
right. Set the readOnly attribute for this property